### ISSUE TITLE
Support parsing "rgb" from PCD files

### ISSUE THREAD

**Issue by abellgithub** [PDAL/PDAL#4456]
PCD contains special handling for Red, Green and Blue stored as a single entity. These may be stored as 4 byte floating or integral values. PDAL should parse the "rgb" field into Red, Green and Blue dimensions.

See: #2880

**Comment by claumeh** [comment]
Yes, I agree. There is a non standard pcd being created with the writers.pcd containing no rgb values. Even I try to tell the writers.pcd what type and order of output I need.

`    {
      "type": "writers.pcd",
      "filename": "data/output.pcd",
      "compression": "ascii",
      "order": "X,Y,Z,Red,Green,Blue"
    },`

Am I doing something wrong? But even opening it in Cloud Compare shows there is no RGB values.

**Comment by claumeh** [comment]
Also open3d confirms that the colors are not contained as I tested it here:

`import open3d as o3d
import numpy as np

# Load the PCD file
pcd = o3d.io.read_point_cloud(input_pcd)  # Replace with the correct path to your PCD file

# Check if the PCD file has color information
if not pcd.has_colors():
    print("PCD file does not contain color information")`

**Comment by pauldeee** [comment]
If I want to read from .pcd and write to .las while retaining the color is there currently no way to accomplish this?

**Comment by abellgithub** [comment]
> You can also do the reverse transformation while reading the .pcd with ferry and assign.
